lib/ftu/onboarding/steps.dart

```diff
@@ -0,0 +1,135 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_animator/flutter_animator.dart';
+
+import '../../common/circle_indicators.dart';
+import '../../common/format_text.dart';
+import '../../common/global_header.dart';
+import '../../common/primary_button.dart';
+import '../../constants/common_constants.dart';
+import '../../constants/ftu_constants.dart';
+import '../login_create_account/login_create_account.dart';
+
+class OnboardingSteps extends StatefulWidget {
+ 
+  OnboardingSteps({Key key}) : super(key: key);
+
+  @override
+  State<StatefulWidget> createState() => _OnboardingSteps();
+}
+
+class _OnboardingSteps extends State<OnboardingSteps> {
+
+  int stepIndex;
+  double screenWidth, screenHeight;
+
+  @override
+  void initState() {
+    super.initState();
+
+    stepIndex = 0;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    screenWidth = MediaQuery.of(context).size.width;
+    screenHeight = MediaQuery.of(context).size.height;
+
+    return Scaffold(
+      backgroundColor: Colors.white,
+      appBar: buildHeader(),
+      body: Container(
+        child: Stack(
+          alignment: Alignment.center,
+          children: [ 
+            buildSlidingContent(),
+            buildFooter()
+          ]
+        )
+      )
+    );
+  }
+
+  GlobalHeader buildHeader() {
+    var isLastStep = stepIndex == onboardingStepsConfig.length - 1;
+    return isLastStep 
+      ? GlobalHeader()
+      : GlobalHeader(
+        actionText: skipAction, 
+        onActionTap: getSkipPage
+      );
+  }
+
+  void getSkipPage() {
+    Navigator.pushReplacement(
+      context,
+      MaterialPageRoute(builder: (context) => LoginCreateAccount())
+    );
+  }
+
+  Widget buildSlidingContent() {
+    return SlideInRight(
```

Comment:
How does it look when we use the `FadeInRight`?

Reply:
Here is what `FadeInRight` looks like:

<img src="https://user-images.githubusercontent.com/25041263/88758968-ee962080-d126-11ea-805b-8802b8207900.gif" width="300"/>

I think `SlideInRight` looks better? What do you guys think? @HelenGezahegn @xiangxinyue @nayancprakash